src/components/ForecastDetails.js

```diff
@@ -1,43 +0,0 @@
-import React from "react";
-import PropTypes from "prop-types";
-import "../styles/ForecastDetails.css";
-
-function ForecastDetails({ forecast }) {
-  const { date, humidity, temperature, wind } = forecast;
-  const formattedDate = new Date(date).toDateString();
-  return (
-    <div className="forecast-details">
-      <div className="forecast-details_date">Date:{formattedDate}</div>
-      <div className="forecast-details_humidity">Humidity:{humidity}</div>
-      <div className="forecast-details_temperaturemax">
-        {" "}
-        Max Temp:
-        {temperature.max}&deg;C
```

Comment:
With some special characters, like the degrees symbol for temperature you can use HTML escape characters. This one for degrees &#176; will work and you'd just use it like you would use any other character